### ISSUE TITLE
Regression test under openjdk-jdk10/test/jdk failed against openjdk10-openj9 sdk 

### ISSUE THREAD

**Issue by sophia-guo** [AdoptOpenJDK/openjdk-tests#382]
Running a simple test and got:
```/team/sophiag/regression/jtreg/jtreg/bin/jtreg -verbose:all -jdk:/team/sophiag/regression/openjdk10j9binary/jdk-10+46/ /team/sophiag/regression/openjdk10/openjdk-jdk10/test/jdk/java/math/BigDecimal/AddTests.java
Directory "JTwork" not found: creating
Directory "JTreport" not found: creating
Exception in thread "main" java.lang.UnsatisfiedLinkError: sun/hotspot/WhiteBox.registerNatives()V
   at sun.hotspot.WhiteBox.<clinit>(WhiteBox.java:66)
   at requires.VMProps.<clinit>(VMProps.java:52)
   at java.lang.Class.forNameImpl(java.base@10-adoptopenjdk/Native Method)
   at java.lang.Class.forName(java.base@10-adoptopenjdk/Class.java:300)
   at com.sun.javatest.regtest.agent.GetJDKProperties.run(GetJDKProperties.java:63)
   at com.sun.javatest.regtest.agent.GetJDKProperties.main(GetJDKProperties.java:46)
Error: failed to get JDK properties for /team/sophiag/regression/openjdk10j9binary/jdk-10+46/bin/java ; exit code 1 

```

**Comment by sophia-guo** [comment]
Suggested by @leonard and @Christian Tornqvist the workround is remove the the requires part in TEST.ROOT.

**Comment by sophia-guo** [comment]
For this workaround we probably need to exclude some jdk/test, which requires some System properties. That means we should never run those tests against openj9.

**Comment by sophia-guo** [comment]
The patch was also merged into https://github.com/ibmruntimes/openj9-openjdk-jdk10 @ShelleyLambert 

The problem is the patch is for jtreg test framework merging into internal patch repo won't help. This is also the reason we add the patch in adopt openjdk-tests repo. Another issue is probably the file https://github.com/ibmruntimes/openj9-openjdk-jdk10/blob/openj9/test/hotspot/jtreg/TEST.ROOT should also be updated as developer did get this issue running sandbox repo tests.

**Comment by andrew-m-leonard** [comment]
Not sure I understand why you're still having a problem...? please explain in more detail? What are you building/testing?
I don't understand this statement: "patch is for jtreg test framework merging into internal patch repo won't help"

**Comment by smlambert** [comment]
In my understanding of this issue, "won't help" probably should read, "seems like a sub-optimal approach".  Better would be to make the jtreg change upstream in openjdk.  All non-hotspot implementations and future OpenJ9 Java versions would then get the benefit of the change, and potentially also benefit developers who are pulling from OpenJDK sandbox repos/forests.

Adopt test builds currently pull a precompiled version of the jtreg executable, as the compilation of the SDK does not package up and make available jtreg (or any test related artifacts) as part of a release. Changes to jtreg in the internal repo are not currently available to us, which means we have temporarily hacked the Adopt openjdk10_j9 test build to force a change.

**Comment by andrew-m-leonard** [comment]
I think the update to TEST.ROOT is quite suitable. The out-of-the-box openjdk TEST.ROOT is configured to provide Hotspot VM test "stat" collection, it seems reasonable that any other VM provides their own TEST.ROOT configuration and anything equivalent. It's not a Jtreg, it's purely test setup. The Adopt test build should provide its own TEST.ROOT config, or patch that provided by openjdk.

**Comment by smlambert** [comment]
Right, I think I misunderstood that there was additional jtreg changes.

**Comment by sophia-guo** [comment]
Fixed in #383
